src/data/resolvers/motions.ts

```diff
@@ -198,6 +208,33 @@ export const motionsResolvers = ({
         }
       }
 
+      if (
+        motionNetworkState === NetworkMotionState.Finalizable ||
+        motionNetworkState === NetworkMotionState.Finalized
+      ) {
+        const newestVoteRevealed = sortedEvents.find(
+          (event) =>
+            event.name === ColonyAndExtensionsEvents.MotionVoteRevealed,
+        );
+        if (newestVoteRevealed) {
+          if (motion.votes[0].gte(motion.votes[1])) {
+            systemMessages.push({
+              type: ActionsPageFeedType.SystemMessage,
+              name: SystemMessagesName.MotionHasFailedFinalizable,
+              createdAt: newestVoteRevealed.createdAt,
+            });
+          }
+        }
+      }
+
+      if (motionNetworkState === NetworkMotionState.Failed) {
+        systemMessages.push({
+          type: ActionsPageFeedType.SystemMessage,
+          name: SystemMessagesName.MotionHasFailedNotFinalizable,
+          createdAt: motion.events[1].toNumber() * 1000,
+        });
+      }
```

Comment:
Why is this here? I just reviewed this code #2451 

Was this cherry-picked ?

Reply:
hehe, i create PR from that branch as i needed changes from it :) I added in todo to rebase it before merge so no worry